portal-web/test/functional/com/liferay/portalweb/macros/FormFields.macro

```diff
@@ -372,6 +372,27 @@ definition {
 			value1 = "${validationErrorMessage}");
 	}
 
+	macro editValidationTargetArgument {
+		Pause(locator1 = "1000");
+
+		Click(locator1 = "FormFields#VALIDATION_ARGUMENT");
+
+		AssertClick(
+			key_validationArgument = "${validationArgument}",
+			locator1 = "FormFields#VALIDATION_ARGUMENT_OPTION_TARGET_INPUT",
+			value1 = "${validationArgument}");
+
+		if (isSet(validationInput)) {
+			Type(
+				locator1 = "FormFields#VALIDATION_INPUT",
+				value1 = "${validationInput}");
+		}
+
+		Type(
+			locator1 = "FormFields#VALIDATION_ERROR_MESSAGE",
+			value1 = "${validationErrorMessage}");
+	}
+
```

Comment:
@Jamersonze Why didn’t you use the existing `editValidation ` macro?

Reply:
Because the existing editValidation macro will try to locate validationInput field, which in some cases i've been working on, will be absent thus breaking the test. But in this case, won't be necessary.

Reply:
Understood! Great job then!